lib/services/cacheUtils.js

```diff
@@ -91,10 +97,18 @@ function createDomainEnabledCacheHandler(domain, processValueFn, cache, cacheTyp
 function cacheAndHold(cache, cacheType, cacheKey, retrieveRequestFn, processValueFn, callback) {
     var cachedValue = cache.data[cacheType].get(cacheKey);
 
+    function getCacheEventId() {
+        return cacheType + ':' + cacheKey;
+    }
+
     if (cachedValue[cacheKey]) {
```

Comment:
I see there are no controls on the existence of `cachedValue`. Just wanted to cross-check with you that it will always exist.